modules/os-eap7-launch/added/launch/tx-datasource.sh

```diff
@@ -85,6 +86,10 @@ function inject_jdbc_store() {
                 <communication table-prefix=\"${prefix}\"/>\\
                 <state table-prefix=\"${prefix}\"/>\\
             </jdbc-store>"
+  if [ -n "${2}" ]; then
+    jdbcStore="<coordinator-environment default-timeout=\"${2}\"></coordinator-environment>\\
+            $jdbcStore"
```

Comment:
Doesn't `coordinator-environment` go in a different namespace than all the `jdbc-store`s? Schema [here](https://github.com/wildfly/wildfly/blob/master/transactions/src/main/resources/schema/wildfly-txn_5_0.xsd#L142) seems to suggest it should be under the transactions subsystem, whereas the stores are under the datasource subsystem?

If so, a simple concatenation at the beginning of the datastore list won't work

Reply:
The naming here is less than perfect, but I think I put it in the right place.

The result of this function is put into `##JDBC_STORE##` tag (or marker) in the `standalone-openshift.xml`. You may find the skeleton for it [here](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/modules/sso/config/launch/setup/74/added/standalone-openshift.xml#L534). As you probably noticed, the `##JDBC_STORE##` marker is placed under `<subsystem xmlns="urn:jboss:domain:transactions:5.0">`.

So I think maybe the `##JDBC_STORE##` should be called `##DETAILED_TRANSACTION_SETTINGS##` or something? But I don't want to introduce such change in this PR.

Reply:
It looks good to me! Though the variable name is not really helpful here.

Reply:
Ok, looks good then